tests/Unit/CapitalTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Adyen\Tests\Unit;
+
+class CapitalTest extends TestCaseMock
+{
+    public function testGetCapitalAccount()
+    {
+        // create Checkout client
+        $client = $this->createMockClientUrl('tests/Resources/Capital/get-capital-account.json');
+
+        // initialize service
+        $service = new \Adyen\Service\CapitalApi($client);
+
+        $result = $service->getCapitalAccount();
+        $this->assertEquals(0, $result->getGrants()[0]->getAmount()->getValue());
```

Comment:
Add a little test here to verify the url, look at other tests to see how (eg LEM or BCL)